Header header
float32 xs_x
float32 xs_y
float32 xs_z
float32 xs_sigma
float32 xs_psi
float32 xs_phi
float32 z_xf
float32 z_xr
float32 z_yf
float32 z_yr
float32 z_zf
float32 z_zr
float32 z_za
float32 z_sigma
float32 z_psi
float32 z_phi
float32 u_v
float32 u_phi_dot
float32 u_delta_dot
float32 k_x
float32 k_y
float32 k_z
float32 k_sigma
float32 k_psi
float32 k_phi
float32 p_x
float32 p_y
float32 p_z
float32 p_sigma
float32 p_psi
float32 p_phi